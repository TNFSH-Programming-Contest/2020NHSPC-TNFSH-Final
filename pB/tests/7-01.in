85
35.5 494.9
34.3 276.7
134.3 276.7
107.5 370.6
117.2 501.0
241.6 535.2
323.3 531.5
314.7 487.6
291.6 524.2
223.3 487.6
206.2 374.2
253.8 282.8
308.6 198.6
323.3 246.2
269.6 337.6
269.6 486.4
340.4 425.4
372.1 492.5
408.6 405.9
446.4 463.2
476.9 427.9
498.0 406.0
556.2 410.8
575.7 355.9
520.8 340.1
464.7 398.6
337.9 336.4
400.1 251.1
448.9 265.7
522.0 295.0
583.0 213.3
546.4 147.4
479.4 190.0
396.4 173.0
463.5 18.2
509.8 13.3
494.0 131.6
591.5 69.4
591.5 -0.1
552.5 34.0
540.3 -35.5
602.5 -64.7
592.8 -130.6
502.5 -81.8
513.5 -145.2
450.1 -140.3
459.9 -74.5
363.5 43.8
347.7 -133.0
309.9 -108.6
282.0 -136.7
181.8 -152.5
225.7 -113.5
286.7 -64.7
251.3 -50.1
170.9 -100.1
87.9 -103.8
81.8 -1.3
114.8 8.4
118.4 -55.0
174.5 -55.0
174.5 -12.3
324.5 -19.6
328.2 93.8
396.4 102.3
386.7 123.0
331.8 129.1
240.4 204.7
205.0 301.1
157.4 326.7
178.2 229.1
91.6 236.4
118.4 184.0
187.9 129.1
181.8 29.1
44.0 26.7
46.5 -131.8
-12.0 -136.7
-64.5 -59.9
-36.4 158.4
-5.9 68.2
134.3 74.3
125.7 129.1
28.2 120.6
-18.1 494.9
-10.0 -54.4
