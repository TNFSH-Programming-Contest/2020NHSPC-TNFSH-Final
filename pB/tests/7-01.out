1687014.772
